jaypeasee/national-parkfinder
# fix randomize issue on mainNav button click

## Is this a fix or a feature?
fix

## What does it fix?
The randomize button in MainNav

## Where should the reviewer start?
App.tsx

## How should this be tested?
Open the page and confirm you can still click the random button to get a park

@saraho1123
@jaypeasee
@baileydunning
